### PR TITLE
fix(Wallet Setup): do not show biometry setup view if the device is n…

### PR DESCRIPTION
**Pull request by achtenhagen**
Highlights in this PR:
- Fixes the issue where the biometrics setup view would be presented even if the device itself was not enrolled in biometric authentication.

### CODE REVIEW

**Comment by achtenhagen** [comment]
The `UIDevice` extension sounds like a good place for that. I can add those changes into this PR. As a side note, we should not store an enabled value for biometrics and instead let the system tell the app if it is or not. I think we have a boolean in UserDefaults which violates this.

**Comment by bchrisarriola** [comment]
👍 another thing we can refactor

File: Blockchain/Extensions/UIDevice+Biometrics.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    @available(iOS 11.0, *)
+    init?(type: LABiometryType) {
+        switch type {
+        case .faceID:
+            self.title = "Face ID"
+            self.asset = "IconFaceID"
+        case .touchID:
+            self.title = "Touch ID"
+            self.asset = "IconTouchID"
```
Comment (bchrisarriola) [inline Blockchain/Extensions/UIDevice+Biometrics.swift]: Call `self.init()` here to avoid duplication?

Comment (achtenhagen) [reply Blockchain/Extensions/UIDevice+Biometrics.swift]: This is unfortunately not possible and creating a helper function results in `Property not initialized at super.init call`. I would suggest keeping this duplication for now.

Comment (achtenhagen) [reply Blockchain/Extensions/UIDevice+Biometrics.swift]: This is a great solution and it works just as intended. Nice!

**Pull request merged**